src/pages/get-started/api-security.md

```diff
@@ -7,6 +7,217 @@ This topic describes best practices for [API security](https://owasp.org/www-pro
+```
+
+And in the case when the project is already installed
+```shell
+# Project is not installed yet
+$ php bin/magento setup:install --help | grep backp
+      --backpressure-logger=BACKPRESSURE-LOGGER                                                  Backpressure logger handler
+      --backpressure-logger-redis-server=BACKPRESSURE-LOGGER-REDIS-SERVER                        Redis server
+      --backpressure-logger-redis-port=BACKPRESSURE-LOGGER-REDIS-PORT                            Redis server listen port
+      --backpressure-logger-redis-timeout=BACKPRESSURE-LOGGER-REDIS-TIMEOUT                      Redis server timeout
+      --backpressure-logger-redis-persistent=BACKPRESSURE-LOGGER-REDIS-PERSISTENT                Redis persistent
+      --backpressure-logger-redis-db=BACKPRESSURE-LOGGER-REDIS-DB                                Redis db number
+      --backpressure-logger-redis-password=BACKPRESSURE-LOGGER-REDIS-PASSWORD                    Redis server password
+      --backpressure-logger-redis-user=BACKPRESSURE-LOGGER-REDIS-USER                            Redis server user
+      --backpressure-logger-id-prefix=BACKPRESSURE-LOGGER-ID-PREFIX                              ID prefix for keys
+```
+
+{:.bs-callout-tip}
+The data (request time and the identifier) will be temporarily stored in the redis. In the case of a non-registered user, the identifier will be his external IP address, and his user ID in the case of a registered user ID.
+
+An example of a command that will add a new connection to the Redis server.
+
+Redis server:
+- Host: 195.34.23.5
+- Port: 9345
+- Password: s0M3StR0NgP@SsW0Rd
+- User: SomeUser
+
+```shell
+$ php bin/magento setup:config:set \
+    --backpressure-logger=redis \
+    --backpressure-logger-redis-server=195.34.23.5 \
+    --backpressure-logger-redis-port=9345 \
+    --backpressure-logger-redis-timeout=1 \
+    --backpressure-logger-redis-persistent=persistent \
+    --backpressure-logger-redis-db=3 \
+    --backpressure-logger-redis-password=s0M3StR0NgP@SsW0Rd\
+    --backpressure-logger-redis-user=SomeUser \
+    --backpressure-logger-id-prefix=some_pref
+```
+After the command is executed, the following configuration will be added to the file `app/etc/env.php`
+
+```php
+[
+//...
+    'backpressure' => [
+        'logger' => [
+            'type' => 'redis',
+            'options' => [
+                'server' => '195.34.23.5',
+                'port' => 9345,
+                'timeout' => 1,
+                'persistent' => 'persistent',
+                'db' => '3',
+                'password' => 's0meStr0ngPassw0rd',
+                'user' => 'SomeUser'
+            ],
+            'id-prefix' => 'some_pref'
+        ]
+    ]
+//...
+];
+```
+Now you are ready to enable this functionality.
+You can do this using the console commands:
+
+1. Enable - 1/ disable - 0 rate limiting for placing orders:
+```php
+$ php bin/magento config:set sales/backpressure/enabled 1
+```
```

Comment:
```suggestion
1. Enable `1` or disable `0` rate limiting for placing orders:

    ```php
    $ php bin/magento config:set sales/backpressure/enabled 1
    ```
```

The code blocks under each list item need returns around them and need to be indented.